### ISSUE TITLE
Feature request: Allow CORS headers "Access-Control-Max-Age"

### ISSUE THREAD

**Issue by noomz** [balderdashy/sails#2345]
It would be better if we can set "Access-Control-Max-Age". So client haven't to request the preflight every time they want to post somethings.

**Comment by IanVS** [comment]
Hi, what's the status of this issue?  I see that the labels were removed, but the issue wasn't closed.  Is there a way to accomplish this currently?

**Comment by tjwebb** [comment]
Sorry about that; I removed the old labels and never added new ones

**Comment by sailsbot** [comment]
Thanks for posting, @noomz.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
